twisted/conch/test/test_ckeygen.py

```diff
@@ -81,12 +106,12 @@ def test_saveKey(self):
         base.makedirs()
         filename = base.child('id_rsa').path
         key = Key.fromString(privateRSA_openssh)
-        _saveKey(key, {'filename': filename, 'pass': 'passphrase'})
+        _saveKey(key, {'filename': filename, 'pass': 'passphrase', 'format': 'md5-hex'})
```

Comment:
please try to wrap lines at 79 chars. thanks!

also, maybe this whole dict can be given a name... like `options` so that we can call it like

```
 _saveKey(key, options)
```

and have an idea what that dict represents